### ISSUE TITLE
Квест <<код>>

### ISSUE THREAD

**Issue by kalashnov** [linostardis/master-thesis#1]
- [x] Научиться генерировать данные в Tensorflow
- [ ] Научиться обучать модель в Tensorflow
- [ ] Понять гиперпараметры: activation, learning rate, epoch, batch size
- [ ] Разберемся как здесь делать constrained optimization 
- [ ] Разберемся как делать более стурктурированную модель (не просто sequential), а заодно, что нам тут интересно с точки зрения диплома
